templates/themes/app/content/views/overlay/ng_video.html.twig

```diff
@@ -9,7 +9,7 @@
     <article {{ toolbar_macros.item_params(content, location) }} class="view-type view-type-{{ view_type }} ng-video vl2">
         <i class="icon-play article-icon" aria-hidden="true"></i>
         <figure class="image">
-            {{ video.poster(content, location) }}
+            {{ video.poster(content, location, null, content_fields.title(content)) }}
```

Comment:
Why are those two parameters added?

Reply:
It's alias and alt text for video.poster macro.

Reply:
i would suggest adding some alias, just to be safe.

Reply:
added alias in standard and overlay - ng_video